scripts/secnetperf-helpers.psm1

```diff
@@ -285,7 +285,7 @@ function Start-LocalTest {
         $pinfo.Arguments = $FullArgs
     } else {
         $pinfo.FileName = "bash"
-        $pinfo.Arguments = "-c `"ulimit -c unlimited && LSAN_OPTIONS=report_objects=1 ASAN_OPTIONS=disable_coredump=0:abort_on_error=1 UBSAN_OPTIONS=halt_on_error=1:print_stacktrace=1 $FullPath $FullArgs && echo ''`""
+        $pinfo.Arguments = "-c `"ulimit -c unlimited && LSAN_OPTIONS=report_objects=1 ASAN_OPTIONS=disable_coredump=0:abort_on_error=1 UBSAN_OPTIONS=halt_on_error=1:print_stacktrace=1 $FullPath $FullArgs && echo ""`""
```

Comment:
Pretty sure this broke Linux. But in general, I don't think we need to change all these quotes.

Reply:
oh.... interesting. The motivation was single tick strings in powershell is a string literal (all enclosed variables won't be expanded) and I thought standardizing on double quotes is good, so Just CMD + F (replace all). I can revert this change if it might help.